src/wagtailtrans/models.py

```diff
@@ -101,6 +102,10 @@ class TranslatablePage(Page):
 
     is_creatable = False
 
+    search_fields = Page.search_fields + [
+        FilterField('language_id'),
```

Comment:
As I understand from the [Wagtail Docs](http://docs.wagtail.io/en/v1.8/topics/search/indexing.html#index-relatedfields) it's not possible to filter on a `FilterField` inside a`RelatedField`.

Indexing the `language__code` is possible, so when using ElasticSearch something can be made to have this working as you would expect, I can't tell how this will behave when using the ORM backend.

Perhaps we should take a good look at searching / indexing and multi-language content and implement this as a feature of `wagtailtrans`. For now this is a quick fix, which can also be made in a project itself.